cirq/google/devices/known_devices.py

```diff
@@ -223,3 +224,37 @@ def _json_dict_(self):
 BRISTLECONE_PROTO = create_device_proto_from_diagram(_BRISTLECONE_GRID,
                                                      [gate_sets.XMON],
                                                      _DURATIONS_FOR_XMON)
+
+_SYCAMORE_GRID = """
+-----AB---
+----ABCD--
+---ABCDEF-
+--ABCDEFGH
+-ABCDEFGHI
+ABCDEFGHI-
+-CDEFGHI--
+--EFGHI---
+---GHI----
+----I-----
+"""
+
+_SYCAMORE_DURATIONS_PICOS = {
```

Comment:
@maffoo are these numbers correct?

Reply:
Yes, these look correct for our current calibration.